### PR TITLE
add example client usage to README

### PR DESCRIPTION
**Pull request by cainj13**
Instead of just using the LB overview from core, thought it would be useful to demonstrate how to use the java client libraries on the README.  That's where I usually go first when I'm trying to figure out how to use a library.

### CODE REVIEW

**Comment by cainj13** [comment]
preview of docs here: https://github.com/cainj13/lightblue-client/tree/docs

File: README.md [hunk 1]
```diff
@@ -1,109 +1,70 @@
...
+        config.setDataServiceURI("https://my.lightblue.com/rest/data");
+        config.setCertFilePath("client-cert.pkcs12");
+        config.setCaFilePath("ca-cert.pem");
+        config.setCertPassword("s3cr3t");
+        config.setUseCertAuth(true);
+        config.setCompression(LightblueClientConfiguration.Compression.NONE);
 
-Example workflow to update a single field *with* lightblue:
-![Fast](https://raw.github.com/lightblue-platform/lightblue/master/docs/fast.png)
+* Be sure that the certificate files wind up on the same runtime classpath as the lightblue libraries, since this is the source lightblue will use when it attempts to load the certificates.
```
Comment (dcrissman) [inline README.md]: or file://

**Comment by dcrissman** [comment]
I have no issues with this change, it certainly is more relevant to the client.

@bserdar @jewzaam @paterczm @BVulaj @snehagunta - Any concerns around not using the sort of pre-canned lightblue README?

File: README.md [hunk 2]
```diff
@@ -1,109 +1,70 @@
 [![Coverage Status](https://coveralls.io/repos/lightblue-platform/lightblue-client/badge.png?branch=master)](https://coveralls.io/r/lightblue-platform/lightblue-client?branch=master)
 
-# Books!
+# Whis is lightblue?
```
Comment (BVulaj) [inline README.md]: Typo?

**Comment by BVulaj** [comment]
@dcrissman I prefer this over the canned lightblue README.  I say go through with it.  I think it would also be worthwhile to make all / most of our READMEs project specific rather than just LB boilerplate.

**Comment by jewzaam** [comment]
I'm fine with not using the canned readme.  We did that so we didn't have to maintain specific readme for each of many projects.  Client is a separate thing, it should have some TLC.  And, there isn't any docs for client yet so we really need to start developing them.

**Comment by jewzaam** [comment]
Maybe something we could build on in the lightblue user guide, under the cookbook section?

http://docs.lightblue.io/

**Comment by dcrissman** [comment]
PR for the minor changes mentioned earlier: https://github.com/lightblue-platform/lightblue-client/pull/242

**Pull request merged**